DiscordBot/Commands/EventCommands.cs

```diff
@@ -225,7 +237,7 @@ public async Task RemoveEvent(CommandContext context)
         }
 
         [Command("remove")]
-        public async Task RemoveEvent(CommandContext context, int eventKey)
+        public async Task RemoveEvent(CommandContext context, [Description("Optional key corresponding to event")] int eventKey)
```

Comment:
It's not really optional, you will need to provide the key at some point right? 🙂

Reply:
The description shows how the command works, you can still invoke the command without the event key (and then it will subsequently prompt you for it)

Reply:
I still think saying it's optional is misleading though so I would leave that out 🙂

Reply:
I can easily just remove the "optional" if you think it would make more sense that way though :D